styles.css

```diff
@@ -113,16 +114,287 @@
 
 .main__button {
   display: inline-block;
+  position: relative;
+  left: 8px;
   margin-top: 40px;
-  font: 18px Roboto, arial, helvetica, sans-serif;
+  font: 18px roboto, arial, helvetica, sans-serif;
   color: #fff;
   background: #0056ec;
-  padding: 18px 28px 18px 21px;
+  padding: 18px 30px 15px 23px;
   text-decoration: none;
   border-radius: 4px;
-  transition: opacity  .5s;
+  transition: opacity 1s;
 }
 
-.main__button:hover {
-  opacity: .5;
+.main__button-blic {
+    position: relative;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/HTML_SNN/pullRequest?prid=6320070)